js/samples/04.ai-apps/h.datasource-azureOpenAI/teamsapp.local.yml

```diff
@@ -54,5 +65,5 @@ deploy:
               BOT_ID: ${{BOT_ID}}
               BOT_PASSWORD: ${{SECRET_BOT_PASSWORD}}
               BOT_TENANT_ID: ${{TEAMS_APP_TENANT_ID}}
-              AZURE_OPENAI_ENDPOINT: ${{AZURE_OPENAI_ENDPOINT}}
```

Comment:
should this be passing in `AZURE_OPENAI_DEPLOYMENT` now?

Reply:
@Carter425 I added this - we should put this in the instructions. The sample had it hard-coded, but its a reasonable thing that you would paste along with your endpoint.

Reply:
@therealjohn what is the change that needs to be made to the instructions?